patches/chrome-installer-util-BUILD.gn.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
+index 2cf4027737f73cff9fcd061ed46b6b92f3587e84..5383c6166368ca23128eb44e8549951268f0a4dd 100644
+--- a/chrome/installer/util/BUILD.gn
++++ b/chrome/installer/util/BUILD.gn
+@@ -246,9 +246,15 @@ action("generate_strings") {
+     "$target_gen_dir/installer_util_strings.rc",
+   ]
+ 
++  brand = "$branding_path_component"
++
++  if (brave_chromium_build && !is_official_build) {
++    brand = brand + "-development"
```

Comment:
I don't think this is going to be sufficient because we'll also have `beta`, etc.. why wouldn't we just modify the branding_path_component in the args.gn?

Reply:
Upstream chrome uses two kinds of brand strings - `google_chrome` and `chromium`.
For specific channel support like beta or dev, they handled under `google_chrome`.
So, I think introducing  `brave-development`  or similar name for unofficial build.
We should cleanup/refactoring to use both 'brave` and `brave-development` in other places.

Reply:
ok, I think I understand what you're trying to do here, but I still think we should modify the `branding_path_component` here https://github.com/brave/brave-browser/blob/master/lib/config.js#L51 like we do in muon. It was only commented out as a temporary workaround and I'm not sure that it's a good idea to modify it here

Reply:
Yep, I agree with you.
How about cleanup/refactoring it as a separate PR?
If we change `branding_path_component` in `args.gn`, many places would be broken such as `brave_strings.grd`.

Reply:
can you create an issue for it and I'll remove the "changes requested"?

Reply:
https://github.com/brave/brave-browser/issues/153 is created for it.

Reply:
looks like I can't directly remove my request for changes, so just ignore it :)

Reply:
@bridiver , can you confirm this PR? or should I wait more from other reviewers?